### PR TITLE
Refactored Longevity Stage2

### PR DESCRIPTION
**Pull request by malhotra700**
List of Changes ->
1. Moved stage_2 function to Longevity class in longevity.py.
2. Moved all the helper functions of stage 2 to a new file longevity_helpers.py
3. Moved longevity stage 2 test case to -> tests/e2e/longevity/test_stage2.py
4. Other minor fixes.

Signed-off-by: malhotra700 <anantmal007@gmail.com>

### CODE REVIEW

File: ocs_ci/ocs/longevity_helpers.py [hunk 1]
```diff
@@ -220,7 +218,14 @@ def _multi_pvc_pod_lifecycle_factory(
...
 
     def factory(
-        num_of_pvcs=100, pvc_size=2, bulk=False, namespace="stage-2", measure=True
+        num_of_pvcs=100,
+        pvc_size=2,
+        bulk=False,
+        namespace="stage-2",
+        measure=True,
+        file_name=None,
+        fio_fraction=0.25,
```
Comment (PrasadDesala) [inline ocs_ci/ocs/longevity_helpers.py:227]: can take it in percentage instead?

Comment (malhotra700) [reply ocs_ci/ocs/longevity_helpers.py:227]: Ack, will change it.

File: ocs_ci/templates/app-pods/performance.yaml [hunk 2]
```diff
@@ -5,6 +5,9 @@ metadata:
   name: perf-pod
   namespace: default
 spec:
+  securityContext:
+    runAsNonRoot: true
+    runAsUser: 1000
```
Comment (PrasadDesala) [inline ocs_ci/templates/app-pods/performance.yaml:10]: Please revert this change

Comment (malhotra700) [reply ocs_ci/templates/app-pods/performance.yaml]: Done

**Review by PrasadDesala — DISMISSED** [review]

**Review by Akarsha-rai — APPROVED** [review]

**Comment by malhotra700** [comment]
The stage2 test run https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/13773/
ran for 7 hrs and completed 4 cycles.

**Review by PrasadDesala — APPROVED** [review]

**Review by Akarsha-rai — APPROVED** [review]
